cmd/apiserver/main.go

```diff
@@ -235,7 +237,12 @@ func main() {
 
 				store := inmem.New()
 
-				api, err := handlers.NewAPI(ctx, logger.Sugar(), db, ipam, fflags, store, signalBus)
+				redisClient := redis.NewClient(&redis.Options{
+					Addr: cCtx.String("redis-server"),
+					DB:   cCtx.Int("redis-db"),
```

Comment:
This will make the client use the same DB and key space as the session store. We need to be careful that the keys we store for users don't conflict with the session keys.  Later I make a recommendation to add a key prefix to our keys.. that might be enough to distinguish our keys from the sessions store.. not sure.

Reply:
updated